Geometry/HGCalCommonData/data/dd4hep/testHGCalV14.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<DDDefinition>
+  <open_geometry/>
+  <close_geometry/>
+
+  <IncludeSection>
+    <Include ref="Geometry/CMSCommonData/data/materials.xml"/>
```

Comment:
```suggestion
    <Include ref="Geometry/CMSCommonData/data/materials/2021/v1/materials.xml"/>
```

Reply:
Shouldn't the latest materials.xml be used rather than the obsolete one?

Reply:
@cvuosalo Carl can you approve this PR. I shall modify the xml file in the next PR. It needs other changes as well because of overlap issues.

Reply:
I also need #30654 to be integrates and then I shall make a new PR correcting this scenario as well where I can correct the material file

Reply:
It's fine to fix the materials.xml reference in a later PR.